File: SampleCodeAPI/Model/NamHocModel.cs
namespace SampleCodeAPI.Model
{
    public class NamHocModel
    {
        public int? id { get; set; }
        public int NamHoc { get; set; }
        public string NienHoc { get; set; }
        public bool? Disable { get; set; }
        public int? DeletedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? DeletedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public DateTime? CreatedDate { get; set; }

    }
}

File: Frontend/Angular/src/app/pages/namhoc/model/nam-hoc.model.ts
import { BaseModel } from "src/app/_metronic/core/models/_base.model";

export class NamHocModel extends BaseModel {
    id: number;
    NamHoc: number;
    NienHoc: string;
    CreatedBy: string;
    CreatedDate: Date | string;
    Disable: boolean;

    clear() {
        this.id = 0;
        this.NamHoc = 0;
        this.NienHoc = '';
        this.CreatedBy = '';
        this.Disable = true;
    }
}